### DISCUSSION TITLE
Cycle 25 started

### DISCUSSION THREAD

**Discussion by MwithM** [bisq-network/bisq#5504] [Show and tell]
- The [rate](https://github.com/bisq-network/compensation/issues/850) for compensation requests on Cycle 25 is 2.23USD/BSQ.
- [Deadline](https://calendar.google.com/calendar/embed?src=adfiuiokq9jtac8t9keomc00b0%40group.calendar.google.com&ctz=Europe%2FMadrid) to submit compensation requests for review is 5 June.
- BSQ trading fee has been [updated](url).

**Comment by ghost** [comment]
3 CRs for cycle 25 were submitted during cycle 24, using the rate at that time.  Which rate should they use?  If they need updating it would be best to remind the users to make the change.  The bot is ready to validate cycle 25 requests now.

**Reply by MwithM** [reply]
They should change it. I'm quite sure they won't fight very much to use the rate from Cycle 24, since it's now lower.
